test_scripts/curl/result.json

```diff
@@ -0,0 +1 @@
":"US"}],"services":{"lldp":{"describe":"uCentral","location":"universe"},"ssh":{"port":22}},"uuid":1624968988},"createdTimestamp":1624625478,"devicePassword":"","deviceType":"AP","firmware":"OpenWrt 21.02-SNAPSHOT r16011+45-6fd65c6573 \/ TIP-devel-71a5b63","fwUpdatePolicy":"","lastConfigurationChange":1624968988,"lastConfigurationDownload":1624913857,"lastFWUpdate":1624625478,"location":"","macAddress":"68:21:5f:da:45:6d","manufacturer":"Edgecore ECW5211","notes":"auto created device.","owner":"","serialNumber":"68215fda456d"},{"UUID":1624971785,"compatible":"edgecore_eap102","configuration":{"interfaces":[{"ethernet":[{"select-ports":["WAN*"]}],"ipv4":{"addressing":"dynamic"},"name":"WAN","role":"upstream","services":["lldp"],"ssids":[{"bss-mode":"ap","encryption":{"ieee80211w":"optional","key":"something","proto":"psk"},"name":"ssid_wpa_2.4g","wifi-bands":["2G"]}]}],"metrics":{"health":{"interval":120},"statistics":{"interval":120,"types":["ssids","lldp","clients"]}},"radios":[{"band":"2G","channel-width":20,"country":"US"}],"services":{"lldp":{"describe":"uCentral","location":"universe"},"ssh":{"port":22}},"uuid":1624971785},"createdTimestamp":1624626111,"devicePassword":"","deviceType":"AP","firmware":"OpenWrt 21.02-SNAPSHOT r16011+50-6fd65c6573 \/ TIP-devel-d9d72e2","fwUpdatePolicy":"","lastConfigurationChange":1624971785,"lastConfigurationDownload":1624970860,"lastFWUpdate":1624626111,"location":"","macAddress":"90:3c:b3:9d:69:18","manufacturer":"Edgecore EAP102","notes":"auto created device.","owner":"","serialNumber":"903cb39d6918"},{"UUID":1624881333,"compatible":"edgecore_eap101","configuration":{"interfaces":[{"ethernet":[{"select-ports":["WAN*"]}],"ipv4":{"addressing":"dynamic"},"name":"WAN","role":"upstream","services":["lldp"]},{"ethernet":[{"select-ports":["LAN*"]}],"ipv4":{"addressing":"static","dhcp":{"lease-count":100,"lease-first":10,"lease-time":"6h"},"subnet":"192.168.1.1\/24"},"name":"LAN","role":"downstream","services":["ssh","lldp"],"ssids":[{"bss-mode":"ap","encryption":{"ieee80211w":"optional","key":"OpenWifi","proto":"psk2"},"name":"OpenWifi","role":"downstream","wifi-bands":["2G","5G"]}]}],"metrics":{"health":{"interval":120},"statistics":{"interval":120,"types":["ssids","lldp","clients"]},"wifi-frames":{"filters":["probe","auth"]}},"radios":[{"band":"5G","channel":52,"channel-mode":"HE","channel-width":80,"country":"CA"},{"band":"2G","channel":11,"channel-mode":"HE","channel-width":20,"country":"CA"}],"services":{"lldp":{"describe":"uCentral","location":"universe"},"ssh":{"port":22}},"uuid":1624881333},"createdTimestamp":1624626264,"devicePassword":"","deviceType":"AP","firmware":"OpenWrt 21.02-SNAPSHOT r16011+50-6fd65c6573 \/ TIP-devel-d9d72e2","fwUpdatePolicy":"","lastConfigurationChange":1624881333,"lastConfigurationDownload":1624913857,"lastFWUpdate":1624626264,"location":"","macAddress":"90:3c:b3:6a:e2:23","manufacturer":"EdgeCore EAP101","notes":"auto created device.","owner":"","serialNumber":"903cb36ae223"}]}
```

Comment:
Should this be part of the PR?

Reply:
Nope, deleted that and added it into .gitignore as it clearly shouldn't be here